extensions/positron-python/src/client/common/interpreterPathService.ts

```diff
@@ -111,11 +111,14 @@ export class InterpreterPathService implements IInterpreterPathService {
     ): Promise<void> {
         resource = PythonSettings.getSettingsUriAndTarget(resource, this.workspaceService).uri;
         if (configTarget === ConfigurationTarget.Global) {
-            const pythonConfig = this.workspaceService.getConfiguration('python');
-            const globalValue = pythonConfig.inspect<string>('defaultInterpreterPath')!.globalValue;
-            if (globalValue !== pythonPath) {
-                await pythonConfig.update('defaultInterpreterPath', pythonPath, true);
-            }
+            // --- Start Positron ---
+            // do not update global interpreter path setting via the Select Interpreter dropdown

```

Comment:
When does this happen? When using Select Interpreter without a workspace?

Reply:
Correct! If you use `Python: Select Interpreter` without a workspace, the interpreter selected overrides whatever the user has set.

Reply:
Got it. I wonder if we change this whether it would confuse any VSCode users

Reply:
I don't have much insight into how often this is used, but I did update the setting description to match our use case.